traffic_ops/traffic_ops_golang/cdnfederation/insert.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO federation (
```

Comment:
Why different sql files? We normally have these sql statements as part of the go file.

Reply:
Yeah, and I find that harder to read than if they're in their own file.

Reply:
Interesting but is it worth the change now just because it is harder to read? Why is this file an exception and not others. Also, makes it harder to review too since it looks like cdnfederations.go underwent a lot of change.

Reply:
I mean, I also thought it would be easier for a reviewer to read them in their own files. I don't think putting more lines into the cdnfederations.go file is going to make the changes to it look any smaller, but if you want these to be inline string constants I can do that.